easybuild/easyconfigs/q/qmflows/qmflows-0.12.1-foss-2021a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'PythonBundle'
+
+name = 'qmflows'
+version = '0.12.1'
+
+homepage = 'https://github.com/SCM-NV/qmflows'
+description = """This library tackles the construction and efficient execution 
+of computational chemistry workflows. This allows computational chemists to use 
+the emerging massively parallel compute environments in an easy manner and focus 
+on interpretation of scientific data rather than on tedious job submission 
+procedures and manual data processing.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = {
+    ('Python', '3.9.5'),
+    ('RDKit', '2021.03.4'),
+    ('HDF5', '1.10.7'),
+    ('SciPy-bundle', '2021.05'),
+    ('Rust', '1.52.1')
+}
+
+use_pip = True
+
+exts_list = [
+    ('more-itertools', '9.0.0', {
+        'checksums': ['5a6257e40878ef0520b1803990e3e22303a41b5714006c32a3fd8304b26ea1ab'],
+    }),
+    ('dill', '0.3.6', {
+        'checksums': ['e5db55f3687856d8fbdab002ed78544e1c4559a130302693d839dfe8f93f2373'],
+    }),
+    ('plams', '1.5.1', {
+        'modulename': 'scm.plams',
+        'checksums': ['5e77632be84a8c12172eb426589cb80187b5b31325d917bdbeb0798f9c5aed46'],
+    }),
+    ('pyparsing', '3.0.9', {
+        'checksums': ['2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb'],
+    }),
+    ('PyYAML', '6.0', {
+        'modulename': 'yaml',
+        'checksums': ['68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2'],
+    }),
+    ('filelock', '3.8.0', {
+        'checksums': ['55447caa666f2198c5b6b13a26d2084d26fa5b115c00d065664b2124680c4edc'],
+    }),
+    ('packaging', '20.9', {
+        'checksums': ['5b327ac1320dc863dca72f4514ecc086f31186744b84a230374cc1fd776feae5'],
+    }),
+    ('graphviz', '0.19.2', {
+        'checksums': ['7c90cebc147c18bcdffcd3c76db58cbface5d45fe0247a2f3bfb144d32a8c77c'],
+        'sources': {'filename': '%(name)s-%(version)s.zip'}
+    }),
+    ('grpcio', '1.50.0', {
+        'modulename': 'grpc',
+        'checksums': ['12b479839a5e753580b5e6053571de14006157f2ef9b71f38c56dc9b23b95ad6'],
+    }),
+    ('protobuf', '4.21.9', {
+        'modulename': 'google.protobuf',
+        'checksums': ['61f21493d96d2a77f9ca84fefa105872550ab5ef71d21c458eb80edcf4885a99'],
+    }),
+    ('grpcio-tools', '1.50.0', {
+        'modulename': 'grpc_tools',
+        'checksums': ['88b75f2afd889c7c6939f58d76b58ab84de4723c7de882a1f8448af6632e256f'],
+    }),
+    ('cryptography', '38.0.3', {
```

Comment:
`cryptography` is already provided by the base Python easyconfig. If the reason to add this newer version is `pyOpenSSL`, I suggest to downgrade pyOpenSSL to version `20.0.1`, which is already used in other easyconfigs in this toolchain and supports the version of `cryptography` in the easyconfig of Python.